README.md

```diff
@@ -1 +1,97 @@
+Main features of this website are:
+
+- Users can see details of all the cars.
+- Users can reserve a car for a special day and in the specific city.
+- Users can add their own cars.
+
+## Built With
+
+- Programming Languages: HTML, CSS, JavaScript.
+- Frameworks: React, Redux and Bootstrap.
+- Code Editor: VS Code.
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql, Rspec, Capybara, Robocop
+
+## Live Demo
+
+[Live Demo Link]()
+
+Kanban Board:
+Kanban Board [link](https://github.com/jahongiry/book-appointment_back-end/projects/1)
+**Initial state of the Kanban Board:**
+<img width="1786" alt="Screen Shot 2022-10-27 at 12 49 58 PM" src="https://user-images.githubusercontent.com/91022355/198223204-f10d5088-a286-4f6f-9d34-fd5f414c9560.png">
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- Have `visual-studio code` or any other code editor installed on your computer.
+
+### Setup
+
+- In order to get a copy of this project you need to download it from https://github.com/jahongiry/book-appointment_front-end.git
+- Extract the zipped file and open it in your code editor
+
+### Install and start
+
+- Run the command bellow in your terminal to get all required files
+
+```
+npm install
+```
+
+```
+npm start
+```
+
+## Authors
+
+👤 **Jahongir Yusupov**
+
+- GitHub: [@githubhandle](https://github.com/jahongiry)
+- Twitter: [@twitterhandle](https://twitter.com/jahongir13)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jahngir-yusupov/)
+
+👤 **Zelalem Fissha**
+
+- GitHub: [@githubhandle](https://github.com/Zelalem1222)
+- Twitter: [@twitterhandle](https://twitter.com/Zelalem52236790)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zelalem-fissha-52b093231/)
+
+👤 **Assad Isah**
+
+- GitHub: [@githubhandle](https://github.com/nottherealalanturing)
+- Twitter: [@twitterhandle](https://twitter.com/AssadIsah)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/AssadIsah/)
+
+👤 **Emirjeta V**
+
+- GitHub: [@githubhandle](https://github.com/myelin0)
+- Twitter: [@twitterhandle](https://twitter.com/jahongir13)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emirjeta-veisllari/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hats off to the whole microverse community
+- Original design idea by [Murat Korkmaz on Behance.](https://www.behance.net/muratk)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

 ## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪